chat-android/src/test/java/com/ably/chat/room/RoomFeatureSharedChannelTest.kt

```diff
@@ -89,16 +98,17 @@ class RoomFeatureSharedChannelTest {
             "clientId",
             createMockLogger(),
         )
-        Assert.assertFalse(room.options.presence!!.enableEvents)
-        Assert.assertTrue(room.options.occupancy!!.enableEvents)
+        Assert.assertFalse(room.options.presence.enableEvents)
+        Assert.assertTrue(room.options.occupancy.enableEvents)
 
         Assert.assertEquals(1, capturedChannelOptions.size)
 
         // CHA-PR9c2 - Check for set presence modes, presence_subscribe flag doesn't exist
-        Assert.assertEquals(3, capturedChannelOptions[0].modes.size)
+        Assert.assertEquals(4, capturedChannelOptions[0].modes.size)
         Assert.assertEquals(ChannelMode.publish, capturedChannelOptions[0].modes[0])
         Assert.assertEquals(ChannelMode.subscribe, capturedChannelOptions[0].modes[1])
         Assert.assertEquals(ChannelMode.presence, capturedChannelOptions[0].modes[2])
+        Assert.assertEquals(ChannelMode.annotation_publish, capturedChannelOptions[0].modes[3])
```

Comment:
Can you add `buildRoomOptions` with `rawMessageReactions=true`, 
We can assert on 
`Assert.assertEquals(ChannelMode.annotation_subscribe, capturedChannelOptions[0].modes[4])`